.circleci/setup.sh

```diff
@@ -70,6 +70,15 @@ if ! type bioconda-utils 2> /dev/null || [[ $BOOTSTRAP == "true" ]]; then
     $WORKSPACE/miniconda/bin/conda install -y git pip --file https://raw.githubusercontent.com/bioconda/bioconda-utils/$BIOCONDA_UTILS_TAG/bioconda_utils/bioconda_utils-requirements.txt
     $WORKSPACE/miniconda/bin/pip install git+https://github.com/bioconda/bioconda-utils.git@$BIOCONDA_UTILS_TAG
 
+    if [[ $OSTYPE == darwin* ]]; then
```

Comment:
@dpryan79, I cherry-picked this into https://github.com/bioconda/bioconda-recipes/pull/19610.

Reply:
Thanks, hopefully that will solve things :)